ui/page/seedbackup/save_seed.go

```diff
@@ -173,13 +186,76 @@ func (pg *SaveSeedPage) Layout(gtx C) D {
 						}),
 					)
 				}),
+				layout.Flexed(1, pg.hexLayout),
 			)
 		},
 	}
 
 	return container(gtx, *pg.Theme, sp.Layout, pg.infoText, pg.actionButton)
 }
 
+func (pg *SaveSeedPage) hexLayout(gtx layout.Context) layout.Dimensions {
+	pg.handleCopyEvent(gtx)
+	card := decredmaterial.Card{
+		Color: pg.Theme.Color.Gray4,
+	}
+
+	return decredmaterial.LinearLayout{
+		Width:       decredmaterial.MatchParent,
+		Height:      decredmaterial.WrapContent,
+		Orientation: layout.Vertical,
+		Background:  pg.Theme.Color.Surface,
+		Border:      decredmaterial.Border{Radius: decredmaterial.Radius(8)},
+		Margin:      layout.Inset{Top: values.MarginPadding0, Bottom: values.MarginPadding16},
+		Padding:     layout.Inset{Top: values.MarginPadding5, Right: values.MarginPadding16, Bottom: values.MarginPadding16, Left: values.MarginPadding16},
+	}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			label := pg.Theme.Label(values.TextSize14, "Seed hex")
+			label.Color = pg.Theme.Color.GrayText1
+
+			return label.Layout(gtx)
+		}),
+		layout.Rigid(func(gtx C) D {
+			return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					card.Radius = decredmaterial.CornerRadius{TopRight: 0, TopLeft: 8, BottomRight: 0, BottomLeft: 8}
+
+					return card.Layout(gtx, func(gtx C) D {
+						return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+							seedString := pg.seed
+							if seedString != "" {
+								hexString, _ := components.SeedWordsToHex(pg.seed)
+								pg.hexLabel.Text = hexString
+							}
+							return pg.hexLabel.Layout(gtx)
+
+						})
+					})
+				}),
+				layout.Rigid(func(gtx C) D {
+					card.Radius = decredmaterial.CornerRadius{TopRight: 8, TopLeft: 0, BottomRight: 8, BottomLeft: 0}
+					return layout.Inset{Left: values.MarginPadding1}.Layout(gtx, func(gtx C) D {
+						return card.Layout(gtx, pg.copy.Layout)
+					})
+				}),
+			)
+		}),
+	)
+}
+
+func (pg *SaveSeedPage) handleCopyEvent(gtx layout.Context) {
+	if pg.copy.Clicked() {
+		clipboard.WriteOp{Text: pg.hexLabel.Text}.Add(gtx.Ops)
+
+		pg.copy.Text = "Copied!"
+		pg.copy.Color = pg.Theme.Color.Success
+		time.AfterFunc(time.Second*3, func() {
+			pg.copy.Text = "Copy"
+			pg.copy.Color = pg.Theme.Color.Primary
+		})
```

Comment:
```suggestion
		time.AfterFunc(time.Second*3, func() {
			pg.copy.Text = "Copy"
			pg.copy.Color = pg.Theme.Color.Primary
			pg.RefreshWindow()
		})
```

After 3 secs you need to refresh the window for the changes to be displayed.